src/tools/cephfs/top/cephfs-top

```diff
@@ -1121,6 +1212,8 @@ if __name__ == '__main__':
         if args.selftest:
             ft.selftest()
             sys.stdout.write("selftest ok\n")
+        elif args.dump:
+            ft.dump_values_to_stdout()
```

Comment:
This looks like a oneshot dump. It would also be useful to continuously dump the json so that test programs can capture stdout for verification.

Reply:
No it's not. It's a continuous dump and it dumps until the user presses Ctrl+C to exit.

Reply:
ACK. In that case does it make sense to add an option to dump it oneshot and exit? I find that useful at times.